data/korath/korath weapons.txt

```diff
@@ -53,6 +54,64 @@ effect "grab-strike impact"
 	"velocity scale" -.1
 
 
+outfit "Shunt-Strike Turret"
+	category "Turrets"
+	cost 821000
+	thumbnail "outfit/shunt-strike"
+	"mass" 42
+	"outfit space" -42
+	"weapon capacity" -42
```

Comment:
Should the Shunt-Strike also receive a reduced weapon capacity usage to account for its asteroid scanner?

Reply:
Hmm.  Does this weapon's ability to target asteroids come from shipboard scanning, or does the miniature swarm of microbots seek out asteroids on their own?